src/commands/MigrationRunCommand.ts

```diff
@@ -73,6 +74,12 @@ export class MigrationRunCommand implements yargs.CommandModule {
                     // noop
             }
 
+            const schemaBuilder = connection.driver.createSchemaBuilder();
+
+            if (schemaBuilder instanceof RdbmsSchemaBuilder) {
+                await schemaBuilder.createMetadataTableIfNecessary();
+            }
```

Comment:
can we make same inside MigrationExecutor#executePendingMigrations ? Near "await this.createMigrationsTableIfNotExist(queryRunner);" line to make sure its consistent and works not only when called from the CLI

Reply:
I guess need to remove this code now?